core/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright 2015 The Apache Software Foundation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0"
+           xsi:schemaLocation="
+             http://www.osgi.org/xmlns/blueprint/v1.0.0 http://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+             http://aries.apache.org/schemas/blueprint-cm/blueprint-cm-1.1.0.xsd
+             ">
+
+<!--    <cm:property-placeholder persistent-id="org.apache.brooklyn.properties" update-strategy="reload">
+        <cm:default-properties>
+            <cm:property name="some-property" value="${karaf.data}/some-path"/>
+        </cm:default-properties>
+    </cm:property-placeholder>-->
+
+    <bean id="localManagementContextService"
+          class="org.apache.brooklyn.core.mgmt.internal.LocalManagementContext" />
```

Comment:
Doesn't look like this will instantiate a valid LocalManagementContext, won't be initialized properly, will be different from the one in `BrooklynLauncher`?